installation/single_machine_docker.rst

```diff
@@ -5,24 +5,44 @@ Docker
 
 This section describes setting up a Citus cluster on a single machine using docker-compose.
 
-**1. Install docker and docker-compose**
+1. Install Docker Engine and docker-compose
+-------------------------------------------
 
-Follow the installation instructions for `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_ on your platform.
+Mac / Windows
+~~~~~~~~~~~~~
 
-**2. Start the Citus Cluster**
+* Install the `Docker Platform <https://www.docker.com/products/overview#/install_the_platform>`_.
+* Start Docker by clicking on the application's icon.
+
+Linux
+~~~~~
+
+.. code-block:: bash
+
+  curl -sSL https://get.docker.com/ | sudo bash
+  sudo usermod -aG docker $USER && exec su -l $USER
+  sudo systemctl start docker
```

Comment:
In my CentOS test Docker did not start automatically after non-root installation. Adding `systemctrl start` fixed that, and also causes no harm in Ubuntu where the service was already started.

The `systemctl` works for Linux distros that use systemd, which I think is recent versions of all major distros. @jasonmp85 does this seem like a good bet to you?